NomaiGrandPrix/manifest.json

```diff
@@ -4,7 +4,7 @@
   "author": "Zac Bauermeister, David Damschen, Ben Kirzhner, Chris Maines",
   "name": "Nomai Grand Prix",
   "uniqueName": "NomaiGrandPrix",
-  "version": "0.1.0",
```

Comment:
- Thoughts on keeping this a v0? Gives us a bit more flex to break things if people decide to build on top of our APIs.
- Also, thoughts on revving patch number for this since it's just a bug fix?

Reply:
Per discussion in discord, this prevents any possible versioning issues from OWML thinking this is before the already-released version. LGTM!